src/components/Organisms/TodoCard/index.jsx

```diff
@@ -8,6 +8,19 @@ import BREAKPOINT from "../../../variables/breakpoint.js";
 const TodoCard = () => {
   const [taskList, settaskList] = useState([]);
 
+  useEffect(() => {
+    if (!localStorage.getItem("taskData")) {
+      settaskList([]);
+    } else {
+      const data = JSON.parse(localStorage.getItem("taskData"));
+      settaskList([...data]);
```

Comment:
```suggestion
      settaskList(data);
```
新たな配列を作成する必要はないのでこれでも大丈夫です！